scale/scale/local_settings_docker.py

```diff
@@ -104,7 +104,18 @@
 CONFIG_URI = os.environ.get('CONFIG_URI', CONFIG_URI)
 
 # Logging configuration
-LOGGING = LOG_CONSOLE_DEBUG if DEBUG else LOG_CONSOLE_INFO
+LOGGING_LEVEL = os.environ.get('SYSTEM_LOGGING_LEVEL', '') #should only be present in running jobs
```

Comment:
I would remove the comment since that env var will be used for other Scale components, not just jobs.